src/main/java/com/softserve/teamproject/controller/UserProfileController.java

```diff
@@ -36,11 +36,11 @@ public User getUserProfile(Principal principal) {
   }
 
   /**
-   * Get groups from user location
+   * Get groups from the authenticated user location
    * @param principal authorized user
    * @return groups in user location
    */
-  @RequestMapping(value = "/user/location/groups", method = RequestMethod.GET)
+  @RequestMapping(value = "/groups/locations/my", method = RequestMethod.GET)
```

Comment:
I'm not sure about that, as our Product owner said that he didn't like it when we referred to the searched items like that. He wanted us to keep the following structure: "requested items"/"filter"...Any other ideas?